tests/test_ui_timeline_and_lineage.py

```
@@ -0,0 +1,489 @@
+    return vault
+
+
+class TestLineageFromEvergreen:
+    def test_evergreen_traces_back_to_raw_source(self, tmp_path, monkeypatch):
+        vault = _seed_lineage_vault(tmp_path)
+        # Patch out get_note_provenance/traceability/inbound_capture
+        # because those want a fully populated knowledge_index — we
+        # only care about the new ``lineage`` field.
+        from ovp_pipeline.ui import view_models as vm
+        monkeypatch.setattr(vm, "get_note_provenance",
+                            lambda *a, **k: {"original_source_note": None,
```

Ruff (0.15.12)

[warning] Unused lambda argument: `a`

(ARG005)